srcjs/widget.js

```diff
@@ -0,0 +1,110 @@
+import HTMLWidgets from 'htmlwidgets';
```

Comment:
I think removing this line will fix https://github.com/react-R/reactR/pull/22#issuecomment-468939428.

Reply:
@alandipert never mind wasn't thinking I believe we should keep this.  I'll keep looking for a solution.

Reply:
I pushed a workaround because I couldn't get it working either -- not sure what's going on there.